reference/generators/cmake.rst

```diff
@@ -188,3 +188,9 @@ These targets are:
   targets from its ``requires``)
 - Each ``CONAN_LIB`` depends on the direct public dependencies ``CONAN_PKG`` targets of its container package. This guarantees
   correct link order.
+
+Default Targets generated by CMake
+----------------------------------
+
+When a compiler or its version is not detected, the CMake helper generates a default target based on the platform operating system.
```

Comment:
When a compiler or its version is not detected, the CMake helper uses a default generator based on the platform operating system.

Reply:
done!